conformance/src/test/resources/application.conf

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018-2019 ABSA Group Limited
```

Comment:
This will cause a conflict with an (almost) same file included in #129

Reply:
Thanks for spotting it. We'll fix the conflict when it arises.